server/src/lib/score-import/import-types/common/batch-manual/converter.ts

```diff
@@ -432,6 +432,46 @@ export async function ResolveMatchTypeToTachiData(
 			return { song, chart };
 		}
 
+		case "ddrSongHash": {
+			const difficulty = AssertStrAsDifficulty(data.difficulty, game, playtype);
+
+			let chart: ChartDocument | null;
+
+			if (context.version) {
+				chart = await db.anyCharts.ddr.findOne({
+					"data.ddrSongHash": data.identifier,
+					playtype,
+					difficulty,
+					versions: context.version,
+				});
```

Comment:
Shouldn't `songHash` be on a SongDocument instead of a ChartDocument? That makes a lot more sense to me.